### DISCUSSION TITLE
How do I create a NetworkPolicy that allows ingress from the gloo edge?

### DISCUSSION THREAD

**Discussion by Stefan-Balta** [kgateway-dev/kgateway#9866] [Q&A]
If I have a NetworkPolicy in some namespace that blocks all ingress, but allows ingress from the gloo-system namespace, I still don't receive any requests. How is that possible? Don't the gloo pods send requests to the upstream pods?

**Comment by yuval-k** [comment]
The gateway pod does  indeed send the requests to the upstream pods.
If this works without the network policy, i suggest looking at the network policy vendor's documentation for debugging tips.
